parachain/pallets/basic-channel/src/inbound/envelope.rs

```diff
@@ -63,36 +71,34 @@ mod tests {
 	use super::*;
 	use hex_literal::hex;
 
-	const LOG: [u8; 284] = hex!(
+	const LOG: [u8; 251] = hex!(
```

Comment:
Has this changed just because the test data is now shorter?

Reply:
Exactly! I didn't check what made the old log longer 🤔